src/inmanta/module.py

```diff
@@ -450,17 +459,13 @@ def get_installed_version(cls, module_name: str) -> Optional[version.Version]:
         if module_name.startswith(ModuleV2.PKG_NAME_PREFIX):
             raise ValueError("PythonRepo instances work with inmanta module names, not Python package names.")
         try:
-            dist: Distribution = pkg_resources.get_distribution(cls.get_python_package_name(module_name))
+            dist: Distribution = pkg_resources.get_distribution(ModuleV2.get_package_name_for(module_name))
             return version.Version(dist.version)
         except DistributionNotFound:
             return None
         except version.InvalidVersion:
             raise InvalidModuleException(f"Package {dist.project_name} was installed but it has no valid version.")
 
-    @classmethod
-    def get_python_package_name(cls, module_name: str) -> str:
-        return ModuleV2.PKG_NAME_PREFIX + module_name.replace("_", "-")
-
```

Comment:
I'd find this more in its place here over at `ModuleV2` but if you prefer it like this it's fine by me.